### DISCUSSION TITLE
Why delete plasma 。

### DISCUSSION THREAD

**Discussion by wepeng** [0xPolygon/matic.js#292] [Q&A] [answered]
I finded the plasma file by delete.
Who can tell me why?

**Comment by ujjwalgupta94** [comment, marked as answer]
Plasma has been moved to seperate repo starting from v3 - https://github.com/maticnetwork/maticjs-plasma

Here is docs for plasma - https://maticnetwork.github.io/matic.js/docs/plasma/

**Comment by wepeng** [comment]
Ok! Thanks very much.
